src/main/java/CinemaRoomRestService/RestService/api/controller/CinemaRoomController.java

```diff
@@ -0,0 +1,46 @@
+package CinemaRoomRestService.RestService.api.controller;
+
+import CinemaRoomRestService.RestService.api.dto.StatisticsDTO;
+import CinemaRoomRestService.RestService.api.dto.CinemaRoomDTO;
+import CinemaRoomRestService.RestService.api.dto.TokenOfSeatDTO;
+import CinemaRoomRestService.RestService.api.mapper.SeatMapper;
+import CinemaRoomRestService.RestService.service.BookingService;
+import CinemaRoomRestService.RestService.domain.entity.Seat;
+import CinemaRoomRestService.RestService.domain.entity.TokenOfSeat;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Use single class import, wild card in Java import statement is bad

Reply:
Fixed